### PR TITLE
 chore: re-enable k8s v1.12.9

### LINKED ISSUE #1421: Upgrade from 1.12.9 is broken
Repro:

{   "apiVersion": "vlabs",   "properties": {     "orchestratorProfile": {       "orchestratorType": "Kubernetes"     },     "masterProfile": {       "count": 1,       "dnsPrefix": "",       "vmSize": "Standard_D2_v2"     },     "agentPoolProfiles": [       {         "name": "agentpool1",         "count": 3,         "vmSize": "Standard_D2_v2",         "osType": "Linux"       }     ],     "linuxProfile": {       "adminUsername": "azureuser",       "ssh": {         "publicKeys": [           {             "keyData": ""           }         ]       }     },     "servicePrincipalProfile": {       "clientId": "",       "secret": ""     }   } }

from commit aacc7cfea5b37ff4c0e516d69f269b7d03a5ff5e consistently results in: 

```
Error: upgrading cluster: Code="NetworkingInternalOperationError" Message="Unknown network allocation error."
```

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
**Reason for Change**:

#1421 cannot be repro'd as of today at 2PM PST. Re-enabling 1.12.9 as this was most likely an external bug.

**Issue Fixed**:

Fixes #1421 

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -364,10 +364,10 @@ K8S_VERSIONS="
 1.13.6
 1.13.5
 1.13.5-azs
+1.12.9
+1.12.9-azs
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: @rjaini FYI

Comment (rjaini) [reply packer/install-dependencies.sh]: Thx for adding.

Comment (rjaini) [reply packer/install-dependencies.sh]: @CecileRobertMichon : Can you please update AzureStack (https://github.com/Azure/aks-engine/blob/master/docs/topics/azure-stack.md) doc as well to include 1.12.9.

File: pkg/api/common/versions.go [hunk 2]
```diff
@@ -102,25 +102,25 @@ var AllKubernetesSupportedVersions = map[string]bool{
...
+	"1.12.9":         true,
 	"1.13.0-alpha.1": false,
 	"1.13.0-alpha.2": false,
 	"1.13.1":         false,
 	"1.13.2":         false,
 	"1.13.3":         false,
 	"1.13.4":         false,
 	"1.13.5":         true,
-	"1.13.6":         false,
+	"1.13.6":         false, // disabled because of https://github.com/kubernetes/kubernetes/issues/78308
```
Comment (mboersma) [inline pkg/api/common/versions.go]: Thanks for adding a comment here.

File: packer/install-dependencies.sh (hunk 1 above)
Comment (CecileRobertMichon) [reply packer/install-dependencies.sh]: already did https://github.com/Azure/aks-engine/pull/1428/files#diff-c14705e88bd9472ced61a1de53fe1a20R157

**Pull request merged**